app/src/main/java/org/mozilla/focus/session/ui/SessionsSheetFragment.java

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.session.ui;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.content.res.Resources;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewAnimationUtils;
+import android.view.ViewGroup;
+import android.view.animation.AccelerateInterpolator;
+
+import org.mozilla.focus.R;
+import org.mozilla.focus.locale.LocaleAwareFragment;
+import org.mozilla.focus.session.SessionManager;
+import org.mozilla.focus.utils.OneShotOnPreDrawListener;
+
+public class SessionsSheetFragment extends LocaleAwareFragment implements View.OnClickListener {
```

Comment:
\[PMD\] INFO: The class 'SessionsSheetFragment' has a Cyclomatic Complexity of 3 \(Highest = 8\)\.